gossip/blockproc/evmmodule/evm.go

```diff
@@ -69,10 +70,6 @@ func (p *OperaEVMProcessor) evmBlockWith(txs types.Transactions) *evmcore.EvmBlo
 		baseFee = nil
 	}
 
-	prevRandao := common.Hash{}
-	if p.net.Upgrades.Sonic {
-		prevRandao.SetBytes([]byte{1}) // TODO provide pseudorandom data?
-	}
```

Comment:
If you drop this condition, you will break the Opera chain compatibility. We will not be able to synchonize Opera mainnet or testnet anymore, which breaks significant part of our CI.
I recommend:
```
	prevRandao := common.Hash{}
	if p.net.Upgrades.Sonic {
		prevRandao = p.prevRandao
	}
```
Alternatively you need to add similar condition elsewhere.

Reply:
Thanks. I will bring it back.